content/blog/2018/app-metrics-istio/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Application Metrics in Istio
+description: How to send application metrics to Prometheus instance in Istio.
+publishdate: 2018-12-19
+subtitle:
+attribution: Mete Atamel
+twitter: meteatamel
+weight: 78
+keywords: [metrics, prometheus, mixer, envoy, sidecar, traffic management]
```

Comment:
Please remove spaces between the words, and put a - in traffic-management.

Reply:
I removed the spaces. I also added ASP.NET to .spelling. If there's more spelling issues, please let me know where to check for spelling errors. I don't have visibility into that.